nil/tests/cli/cli_test.go

```diff
@@ -81,13 +81,20 @@ func (s *SuiteCliService) TestCliBlock() {
 	block, err := s.DefaultClient.GetBlock(s.Context, types.BaseShardId, 0, false)
 	s.Require().NoError(err)
 
-	res, err := s.cli.FetchBlock(types.BaseShardId, block.Hash.Hex())
+	res, err := s.cli.FetchDebugBlock(types.BaseShardId, block.Hash.Hex(), true, false, true)
```

Comment:
Damn, how ugly this train of boolean flags looks in WinAPI style 🙈 In C++ we had a rule not to use bool parameters in functions other than trivial setters, but to have an enum with two values. I'm not sure how suitable this is for Go with the absence of enums as such and weak typing 😞